tgui/packages/tgui/styles/interfaces/CloningConsole.scss

```diff
@@ -0,0 +1,55 @@
+@use '../colors.scss';
+@use '../components/Button.scss';
+
+.Cloning-Console_FlexTable {
```

Comment:
I believe BEM also uses just lowercase, so this class would be best written as `.cloning-console__flex-table`.